setup.py

```diff
@@ -17,8 +17,7 @@
 
 requirements = [
         "bitarray-hardbyte>=0.8.0",
-        "cffi>=1.7",
-        "clkhash>=0.15.0",
+        "clkhash>=0.16.0",
```

Comment:
since you removed clkhash from the benchmark, we don't need the clkhash dependency here any more. Also, clkhash 0.15.0 works just fine with your code.

Reply:
better move it to the tests_require section.